### ISSUE TITLE
Company Communicator Initial Deployment Failed

### ISSUE THREAD

**Issue by Khiron91** [OfficeDev/microsoft-teams-apps-company-communicator#1152]
Have tried now to setup the company communicator (fresh, no upgrade) to a customer but it failed via Powershell and also via manual deployment

Powershell:
![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/124042836/782c9ea6-5d1f-41e7-a4f0-bcbb650e5611)

Manual Deployment:
![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/124042836/cb7c5832-b4df-4e8e-b6b1-624b6c6ecbc5)

**Comment by Khiron91** [comment]
have deleted the web app + changed the web GUIDs and then it worked. still it would be good to know why the posershell version isn´t working with the latest azure cli version

**Comment by gsv022** [comment]
Hi @Khiron91 ,
Thanks for raising the query. The present code is compatible with  Azure CLI 2.30.0.  After 2.30.0 of CLI i.e., >2.30.0 version the parameter(--available-to-other-tenants) has been changed. This is the reason for updated version of CLI it shows as unrecognized arguments. If the parameter has been updated with latest one then the previous version of CLI will start breaking.

So the workaround is to go for uninstalling the latest version of CLI and install 2.30.0 CLI version then go for PowerShell deployment. We will be updating the PowerShell script  and it will be part of upcoming release soon.

Hope I conveyed.
